torchinductor/codecache.py

```diff
@@ -10,6 +10,7 @@
 import tempfile
 import types
 from ctypes import cdll
+from filelock import Timeout, FileLock
```

Comment:
From the docs:

```
All log messages by this library are made using the DEBUG_ level, under the filelock name. On how to control displaying/hiding that please consult the [logging documentation of the standard library](https://docs.python.org/3/howto/logging.html). E.g. to hide these messages you can use:

logging.getLogger("filelock").setLevel(logging.INFO)
```

Not super important, but consider modifying the log level to match whatever is stipulated in config.